CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## v1.0.0-beta3 (unreleased)
+* Remove `Scope` method from the `config.Provider` interface.
```

Comment:
Also this is potentially `[BREAKING]` right, since people could be calling Scope() in their code?